app/config/config.php

```diff
@@ -19,7 +19,7 @@
         'password' => $db['pass'],
         'dbname'   => $db['path'],
         'host'     => $db['host'],
-        'port'     => $db['port'],
+        'port'     => $db['port'] ?? 3306,
```

Comment:
what about adding the `port` array-key only if the `port` is present and leave the default value to the underlying DB layer?